requirements.txt

```diff
@@ -1,3 +1,4 @@
+    pre-commit
```

Comment:
do we want to select a specific version? Are there any experiences if they changes something surprisingly?

Reply:
Yeah, I think specifying a version is better. It's done in [9020d6f](https://github.com/CQCL/pytket/pull/218/commits/9020d6f031bfe595521a7c9e8ab1056effb61e5d)